.travis.yml

```diff
@@ -18,8 +18,8 @@ env:
 
   matrix:
   - LINT_CHECK="1"
-  - TESTS="1" ODOO_REPO="OCA/OCB"
-  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+  - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE='base_tier_validation,base_tier_validation_formula'
+  - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1" EXCLUDE='base_tier_validation,base_tier_validation_formula'
```

Comment:
@mourad-ehm thanks for this module, but probably you also have to add test without your module but with `base_tier_validation,base_tier_validation_formula` or maybe I missed smth ?

Reply:
I used tearDownClass to avoid this bug.